CONTRIBUTING.md

```diff
@@ -42,7 +42,7 @@ To build, use the provided `build.sh` which builds and tests each of the compone
 5. Associate the change with an existing issue or file a [new issue](../../issues).
 6. Create a pull request!
 
-**Note** that you will not be able to run the integration tests for `jib-gradle-plugin` or `jib-maven-plugin` because those push to our integration-testing GCP project. If you would like to run integration tests, change all uses of the `jib-integration-testing` project to your own GCP project and run `./gradlew integrationTest` for `jib-gradle-plugin` and `./mvnw verify -Pintegration-tests` for `jib-maven-plugin`.
+**Note** that in order to run integration tests, you will need to set the environment variable `JIB_INTEGRATION_TESTING_PROJECT` to the GCP project you would like to use for testing. Otherwise, feel free to skip integration tests.
```

Comment:
Maybe we should also mention that they'll need docker installed to run the integration tests.